### ISSUE TITLE
⛔ ♿ Ally is breaking other plugin dialogs and affecting sales

### ISSUE THREAD

**Issue by Kalytrix** [elementor/elementor#33402]
### Description

I have modern tribes event calendar, and Ally is blocking the customer side's ticket input. When ally is off, the modal is complete with fields if included, submit buttons, etc. When Ally is turned on, the modal's title is present, but nothing else, giving no way to add a ticket to the cart. 

### Steps to reproduce

1. Ally off, hello theme on, elementor/elementor pro, modern tribe event calendar & tickets pro, and woocommerce on 
2. Go to an existing event with tickets on the front end
3. Click the + to add one or more tickets
4. click "get tickets" and see the dialog is full 
5. turn ally on in another tab
6. go back to event page, clear cache and try to add a ticket again - the dialog will be empty, with no way to actually add the ticket to the cart

notes - i also tried with and without a template, with and without the modern tribe's Hello Elementor compatibility plugin, without the classic editors, etc. The other thing that ever made the dialog break in any of these situations is turning Ally back on.  This means the issue lies with Ally, not with the third party plugin. 

### Expected behavior

I expect to click on "get tickets" and see the dialog with the buttons to continue

<img width="836" height="284" alt="Image" src="https://github.com/user-attachments/assets/e86ef3bb-22ba-4a9e-a86b-90c7aa5867f3" />

### Elementor System Info

[Redacted by Moderation] 

### Agreement

- [ ] I confirm I have read and followed all the guidelines and instructions outlined in the Elementor Bug Report form.
- [x] I agree that my issue may be closed without further action if it doesn't meet all the requirements outlined in the Elementor Bug Report form.

**Comment by nicholaszein** [comment]
Hi.

If you have a subscription for Ally, please login to your my.elementor.com Account and reach out to Elementor Support. If you don't have one, please go to the free Support Forum in WordPress. This repository is for handling Elementor Editor confirmed Bug Reports only as explained in both the [Bug Report Form](https://elemn.to/gh-new-bug-report), and [Contribution Guidelines](https://elemn.to/gh-contributing). For personal support use the appropriate channels instead.

Thank you.
